RecoJets/Configuration/python/RecoPFJets_cff.py

```diff
@@ -113,10 +94,12 @@
                            fixedGridRhoFastjetCentral,
                            fixedGridRhoFastjetCentralChargedPileUp,
                            fixedGridRhoFastjetCentralNeutral,
+                           puppi,
```

Comment:
please provide some timing measurements for the HI case; about 100 events of wf 140.56 should suffice.
was puppi validated for the HI data?

IIRC, a different "pileup" (underlying event) subtraction scheme is used for HI physics, at least for somewhat central events.

Reply:
@FHead confirmed that we don't use PUPPI for HI RECO, so we can remove this here.

Reply:
an update here (drop puppi from recoPFJetsHITask) is still pending